### DISCUSSION TITLE
How to use inline chat with Copilot?

### DISCUSSION THREAD

**Discussion by orlandoxu** [JetBrains/ideavim#987] [Q&A]
When using Copilot's inline chat, there is a conflict with Ideavim. The backspace key cannot be used in inline chat.

Does anyone know how to solve it?

**Comment by FranciscoDumont** [comment]
Same issue here, very frustrating. 
As a workaround, I'm double-clicking to select the word I want to delete, then typing to replace it.

**Comment by zepocas** [comment]
Also waiting on a fix for this. Very annoying.

**Comment by huyz** [comment]
We gotta keep hassling the GitHub folks, I guess?
Can anyone suggest some other AI tools that are on par or better than  Copilot?

**Comment by dman777** [comment]
I am hitting this also :)

**Comment by sadan4** [comment]
I'm using this workaround, and it has been working fine so far
the bug seems to vanish while in insert mode
```vim
sethandler <C-i> a:vim
nmap <C-i> i<Action>(OpenInlineChatAction)
imap <C-i> <Action>(OpenInlineChatAction)
```

**Reply by huyz** [reply]
Good to know!
